### DISCUSSION TITLE
Ashlands Compatibility

### DISCUSSION THREAD

**Discussion by joshnoe94** [Mydayyy/Valheim-ServerSideMap#65] [Q&A]
Hello, I'm looking into updating my server to Ashlands and making a list of which mods I use are at least mostly working. Does anyone know if this mod will require an update? 

I realize it's not been long since Ashlands released, so I'm not trying to rush anyone :)

**Comment by Mydayyy** [comment]
Greetings,

I haven't actively played Valheim in a long time, I just kept updating the mod whenever I got notified that an update broke is. If anyone can chime in here I would appreciate it. The last update was on April 11th and atleast back then it was fully functional.

If the mod is incompatible with the newest version feel free to file an issue and I will take a look at it and check whether fixing it is feasible.

Best Regards
Mydayyy
